### PR TITLE
No-Wiki History View

### PR DESCRIPTION
**Pull request by codiejay**
# No-Wiki History View History View
_Users now see a "No wiki history" UI when a wiki has just been created and has zero commits._
## Linked issues

closes #298, closes #298, closes https://github.com/EveripediaNetwork/issues/issues/298

### CODE REVIEW

File: src/components/Wiki/History/NoHistoryView.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+export const NoHistoryView = ({ wiki }: NoHistoryViewProps) => {
+  return (
+    <Flex
+      w={{ base: '100%' }}
+      alignItems={{ base: 'center' }}
+      justifyContent={{ base: 'center' }}
+      direction={{ base: 'column' }}
+      mt={{ base: '22' }}
+      mb={{ base: '96' }}
```
Comment (Royal-lobster) [inline src/components/Wiki/History/NoHistoryView.tsx:14]:
```suggestion
      mt={{ base: '12', lg: '24' }}
```
margin bottom is giving too much white space the bottom. i think the page padding is enough for balance. 22 is not in chakra spacing list so its applying 22px. make it 24 so it gives enough space. also added 12 to make the space smaller in mobile devices and to be consistent with margin top on the sentence

File: src/components/Wiki/History/NoHistoryView.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      >
+        <Image
+          flex={1}
+          marginInlineStart="0 !important"
+          src="/images/noWikiHistory.png"
+          w={{ base: '100%', sm: '80%', md: '60%', lg: '15%' }}
+        />
+      </Flex>
+
+      <Box mt={{ lg: '32', base: '12' }} textAlign="center">
```
Comment (Royal-lobster) [inline src/components/Wiki/History/NoHistoryView.tsx:29]:
```suggestion
      <Box mt={{ base: '12', lg: '24' }} textAlign="center">
```
Made this 24 as well to balance the margin top at the picture

**Review by Royal-lobster — CHANGES_REQUESTED** [review]

File: src/components/Wiki/History/NoHistoryView.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export const NoHistoryView = ({ wiki }: NoHistoryViewProps) => {
+  return (
+    <Flex
+      w={{ base: '100%' }}
+      alignItems={{ base: 'center' }}
+      justifyContent={{ base: 'center' }}
+      direction={{ base: 'column' }}
+      mt={{ base: '12', lg: '24' }}
+      mb={{ base: '12' }}
```
Comment (Royal-lobster) [inline src/components/Wiki/History/NoHistoryView.tsx:14]:
```suggestion
```
Margin at the bottom seems to be too much. i think padding of page is enough for space

File: src/components/Wiki/History/NoHistoryView.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import { Flex, Box, Image, Text } from '@chakra-ui/react'
+import React from 'react'
+import { Wiki } from '@/types/Wiki'
+
+export const NoHistoryView = ({ wiki }: NoHistoryViewProps) => {
+  return (
+    <Flex
+      w={{ base: '100%' }}
+      alignItems={{ base: 'center' }}
+      justifyContent={{ base: 'center' }}
```
Comment (anubra266) [inline src/components/Wiki/History/NoHistoryView.tsx:11]: Nothing wrong here, but just so you know, when the component is a `<Flex/>` you can do:
```sugestion
      align={{ base: 'center' }}
      justify={{ base: 'center' }}
```

Comment (anubra266) [inline src/components/Wiki/History/NoHistoryView.tsx:11]: Nothing wrong here, but just so you know, when the component is a `<Flex/>` you can do:
```sugestion
      align={{ base: 'center' }}
      justify={{ base: 'center' }}
```

File: src/components/Wiki/History/NoHistoryView.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      direction={{ base: 'column' }}
+      mt={{ base: '12', lg: '24' }}
+    >
+      <Flex
+        w={{ lg: '45%', base: '100%' }}
+        alignItems={{ base: 'center' }}
+        justifyContent={{ base: 'center' }}
```
Comment (anubra266) [inline src/components/Wiki/History/NoHistoryView.tsx:18]: If you're not having different breakpoints and just `base`, then you don't need the object syntax.
```suggestion
        alignItems="center"
        justifyContent="center"
```

**Review by anubra266 — APPROVED** [review]

**Review by Royal-lobster — APPROVED** [review]

**Pull request merged**